compiler/src/dmd/globals.d

```diff
@@ -223,7 +227,7 @@ extern (C++) struct Param
 
     const(char)[] argv0;                // program name
     Array!(const(char)*) modFileAliasStrings; // array of char*'s of -I module filename alias strings
-    Array!(const(char)*) imppath;       // array of char*'s of where to look for import modules
+    Array!(ImportPathInfo) imppath;       // array of import path information of where to look for import modules
```

Comment:
please update `arraytypes.h` with this

Reply:
I don't know about this, the others such as ``modFileAliasStrings`` should be a ``Strings`` if this was done.

I'm not against somebody doing a follow up PR on this, but I am feeling like its a bit far for this PR.